lib/pbench/server/api/resources/datasets_inventory.py

```diff
@@ -53,27 +53,35 @@ def _get(self, params: ApiParams, _) -> Response:
 
         GET /api/v1/datasets/inventory/{dataset}/{path}
         """
+        try:
+            path = params.uri["path"]
```

Comment:
Adding my own review comment separate from @dbutenhof's to tracke that I'd also like to see this changed to `params.uri.get("path")`.

Also, similar to https://github.com/distributed-system-analysis/pbench/pull/2954/files#r945904587, can we consider a name change to `object` or `target`?

Reply:
I'd be OK with `target`.